pkgs/top-level/all-packages.nix

```diff
@@ -11273,6 +11273,16 @@ with pkgs;
     modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders nginxModules.shibboleth ];
   };
 
+  nginxStableModSecurity-beta = callPackage ../servers/http/nginx/stable.nix {
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders modsecurity-beta ];
```

Comment:
I don't think itis necessary to add specialized nginx version to top level packages. Let users add required modules

Reply:
I agree with @danbst. See [here](https://nixos.org/nixos/manual/#sec-customising-packages) how this can be done.